### ISSUE TITLE
딥러닝과 직관적 이해

### ISSUE THREAD

**Issue by kangthink** [kangthink/kangthink.github.io#3]
발상
- 딥러닝 네트워크에 관한 사고 실험 모델을 하나하나 세워서
- epoch, batch, drop out, normalization, cost function, 
- 작은 단위 용어 [데이터 형태] -> 미리보기 컴포넌트로
- playground 가지고 놀기 [활동 체크하고]
- pytorch, tensorflow 간단히 구현
- 하나는 퍼셉트론과 레이어 관점
- 다른 하나는 matrix 곱
- golang으로 데이터 병렬로? 
- 최종 목표는 딥러닝 이해하고 코드로도 구현 할 수 있도록

**Comment by kangthink** [comment]
계산 그래프 이해하기 글도 연관시킴
